### PR TITLE
added GPS API to insert user location

### PR DESCRIPTION
**Pull request by zarina-m**
included GPS API to generate user's location and inputs it into the location search bar

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -64,6 +64,17 @@ restaurantsApp.getRestaurantIDs = (searchTerm, searchLocation) => {
...
+restaurantsApp.getCity = async function() {
+    // wait for the result to come back (promise)
+    let location = await $.ajax({
+        url: "https://geolocation-db.com/jsonp",
+        jsonpCallback: "callback",
+        dataType: "jsonp"
+    });
+
+    return location.city;
+}
```
Comment (Mila-ashtari) [inline script.js:77]: can you explain why the getcity function is an async function?

File: script.js [hunk 2]
```diff
@@ -99,6 +110,10 @@ restaurantsApp.init = function () {
 }
 
 
-$(function() {
+$(async function() {
     restaurantsApp.init()
+
+    // Get user city and populate the locationTerm input
+    let userCity = await restaurantsApp.getCity();
+    $('#locationInput').val(userCity)
```
Comment (Mila-ashtari) [inline script.js]: can you explain also here why the document ready has became an async function. Also the getcity function should be called in the init function. the only function called in document ready is the init.

**Review by Mila-ashtari — COMMENTED** [review]
the auto location from browser is so cool

File: script.js (hunk 1 above)
Comment (zarina-m) [reply script.js:77]: i wanted to `await` the AJAX call so i could return only the city from `getCity()`. in order to await an AJAX call, we have to make the outer function `async`.

**Pull request merged**